backend/src/main/java/aiversum/backend/rest/ChatController.java

```diff
@@ -77,6 +91,16 @@ public Flux<String> generateOllama(String model, String messages) {
 
         return generateResponse(messages, streamingChatLanguageModel);
     }
+    public Flux<String> generateVertexai(String model, String messages){
+        StreamingChatLanguageModel streamingChatLanguageModel = VertexAiGeminiStreamingChatModel.builder()
+                .project("") /** The PROJECT field represents the variable you set when creating a new Google Cloud project. */
+                .modelName(model)
+                .location("europe-west1")
```

Comment:
This should be moved to application.properties so everyone can pick different location